<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="INBLRNB0299" ignored="0" name="AddNewPayee" tests="5" failures="1" timestamp="30 Nov 2019 12:40:22 GMT" time="7.802" errors="0">
  <testcase name="addNewPayee" time="0.716" classname="tests.AddNewPayee"/>
  <testcase name="addNewPayee" time="0.169" classname="tests.AddNewPayee"/>
  <testcase name="addNewPayee" time="0.138" classname="tests.AddNewPayee"/>
  <testcase name="addNewPayee" time="0.106" classname="tests.AddNewPayee"/>
  <testcase name="test02" time="0.002" classname="tests.AddNewPayee">
    <failure type="java.lang.AssertionError" message="expected [2] but found [1]">
      <![CDATA[java.lang.AssertionError: expected [2] but found [1]
at tests.AddNewPayee.test02(AddNewPayee.java:58)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test02 -->
</testsuite> <!-- AddNewPayee -->
